### DISCUSSION TITLE
Is it possible to search in multi-collections at once?

### DISCUSSION THREAD

**Discussion by iFurySt** [milvus-io/milvus#33380] [Q&A and General discussion]
We split the data into multiple collections based on business, but we'd like to search in a few of the collections at once. Is it possible to do this in Milvus? Or do we need to create a logic layer to search multiple times and combine the results?

**Comment by xiaofan-luan** [comment]
why not use multiple partitions

**Reply by iFurySt** [reply]
Thanks for your reply. I'll give it a try
